enterprise/reporting/reporting.go

```diff
@@ -70,6 +71,13 @@ func NewFromEnvConfig(log logger.Logger) *HandleT {
 	var sleepInterval, mainLoopSleepInterval time.Duration
 	reportingServiceURL := config.GetString("REPORTING_URL", "https://reporting.rudderstack.com/")
 	reportingServiceURL = strings.TrimSuffix(reportingServiceURL, "/")
+
+	sourcesWithEventNameTrackingDisabledEnv := config.GetString("SOURCES_WITH_EVENT_NAME_TRACKING_DISABLED", "")
+	sourcesWithEventNameTrackingDisabled := []string{}
+	if len(sourcesWithEventNameTrackingDisabledEnv) > 0 {
+		sourcesWithEventNameTrackingDisabled = strings.Split(sourcesWithEventNameTrackingDisabledEnv, ",")
+	}
```

Comment:
`GetStringSlice` can be used instead.
```suggestion
	sourcesWithEventNameTrackingDisabled := config.GetStringSlice("SOURCES_WITH_EVENT_NAME_TRACKING_DISABLED", "")
```